entc/gen/type.go

```diff
@@ -1078,6 +1078,14 @@ func (f Field) BuilderField() string {
 	return builderField(f.Name)
 }
 
+// TypeName returns the type name for the field.
+func (f Field) TypeName() string {
+	if n := f.typ.PackageAlias(); n != "" && f.IsEnum() {
+		return strings.Replace(f.Type.String(), f.typ.PackageDir(), f.typ.PackageAlias(), 1)
+	}
+	return f.Type.String()
+}
```

Comment:
Please, see: https://github.com/ent/ent/blob/master/entc/gen/type.go#L981-L982
I believe we can handle this in `aliases`. Check out: https://github.com/ent/ent/blob/master/entc/gen/type.go#L1006-L1035